web/pgadmin/browser/server_groups/servers/databases/schemas/collations/static/js/collation.ui.js

```diff
@@ -22,6 +22,9 @@ export default class CollationSchema extends BaseUISchema {
       lc_type: undefined,
       lc_collate: undefined,
       description: undefined,
+      provider: 'libc',
+      is_deterministic: true,
+      flag_is_deterministic: false,
```

Comment:
What is the use of flag_is_deterministic if we are using is_deterministic already?